functions.php

```diff
@@ -515,3 +515,60 @@ function twentytwenty_read_more_tag() {
 	add_filter( 'the_content_more_link', 'twentytwenty_read_more_tag' );
 
 }
+
+/**
+ * WIP: These should be moved to the Custom class.
+ */
+add_action( 'customize_controls_enqueue_scripts', function() {
+
+	// Add script for color calculations.
+	wp_enqueue_script(
+		'twentytwenty-color',
+		get_template_directory_uri() . '/assets/js/color.js',
+		[ 'wp-color-picker' ],
+		time(), // WIP: Using time() instead of a real version for cache-busting while developing.
+		false
+	);
+
+	// Add script for controls.
+	wp_enqueue_script(
+		'twentytwenty-customizer',
+		get_template_directory_uri() . '/assets/js/customizer.js',
+		[ 'twentytwenty-color', 'customize-controls', 'underscore', 'jquery' ],
+		time(), // WIP: Using time() instead of a real version for cache-busting while developing.
+		false
+	);
+});
+
+/**
+ * WIP: Move these where appropriate (of course using a regular function instead of closures).
+ */
+add_action( 'customize_preview_init', function() {
+	$js_dependencies = array( 'customize-preview', 'jquery' );
+	wp_enqueue_script(
+		'twentytwenty-customizer-preview',
+		get_theme_file_uri( '/assets/js/customizer-preview.js' ),
+		$js_dependencies,
+		time(), // WIP: Using time() instead of a real version for cache-busting while developing.
+		true
+	);
+} );
```

Comment:
Just like the other one above, where do we want this enqueue to be in our files? And again, change `time()` to version.